tests/FakeItEasy.Tests/Core/DefaultFakeManagerAccessorTests.cs

```diff
@@ -115,7 +115,7 @@ public void Should_fail_when_getting_manager_from_object_where_a_wrong_typed_fak
             // Assert
             exception.Should()
                 .BeAnExceptionOfType<ArgumentException>()
-                .WithMessage("The specified object is not recognized as a fake object.");
+                .WithMessage($"Object '{proxy.ToString()}' of type '{proxy.GetType().FullNameCSharp()}' is not recognized as a fake object.");

```

Comment:
… because you made the comment in the same place, @thomaslevesque?

I think you meant down on line 118.